pysradb/cli.py

```diff
@@ -604,32 +679,129 @@ def parse_args(args=None):
     subparser.add_argument(
         "--use_ascp", "-a", action="store_true", help="Use aspera instead of wget"
     )
-    subparser.add_argument(
-        "--col", help="Specify column to download", default="sra_url"
-    )
+    subparser.add_argument("--col", help="Specify column to download")
     subparser.add_argument(
         "--threads", "-t", help="Number of threads", default=1, type=int
     )
     subparser.set_defaults(func=download)
 
-    subparser = subparsers.add_parser("search", help="Search SRA for matching text")
-    subparser.add_argument("--saveto", help="Save metadata dataframe to file")
-    subparser.add_argument("--db", help="Path to SRAmetadb.sqlite file", type=str)
+    # pysradb search
+    subparser = subparsers.add_parser("search", help="Search SRA/ENA for matching text")
     subparser.add_argument(
-        "--assay", action="store_true", help="Include assay type in output"
+        "-o", "--saveto", help="Save search result dataframe to file"
     )
     subparser.add_argument(
-        "--desc", action="store_true", help="Should sample_attribute be included"
+        "-s",
+        "--stats",
+        action="store_true",
+        help="Displays some useful statistics for the search results.",
+    )
+    subparser.add_argument(
+        "-g",
+        "--graphs",
+        nargs="?",
+        const="all",
+        help=(
+            "Generates graphs to illustrate the search result. "
+            "By default all graphs are generated. \n"
+            "Alternatively, select a subset from the options below in a space-separated string:\n"
+            "daterange, organism, source, selection, platform, basecount"
+        ),
     )
     subparser.add_argument(
-        "--detailed", action="store_true", help="Display detailed metadata table"
+        "-d",
+        "--db",
+        choices=["ena", "geo", "sra"],
+        default="sra",
+        help="Select the db API (sra, ena, or geo) to query, default = sra",
     )
     subparser.add_argument(
-        "--expand", action="store_true", help="Should sample_attribute be expanded"
+        "-v",
+        "--verbosity",
+        choices=[0, 1, 2, 3],
+        default=2,
+        help="Level of search result details (0, 1, 2 or 3), default = 2",
```

Comment:
Can you elaborate o what each of the verbosity level reflect in short? For example 0: experiment and run accessions only 2: records in 1 + pmids 3: ...

Reply:
Resolved in 7a89efd